.changeset/selfish-pumas-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-incremental-ingestion': patch
+---
+
+Use new backend feature api to customize `rootConfig`.
```

Comment:
Oh no need for this changeset. The action might complain but it's not able to detect that `run.ts` is only for local dev